tests/lib/rules/return-from-computed.js

```diff
@@ -0,0 +1,59 @@
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/return-from-computed');
+const RuleTester = require('eslint').RuleTester;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const eslintTester = new RuleTester();
+eslintTester.run('return-from-computed', rule, {
+  valid: [
+    {
+      code: 'let foo = computed("test.length", { get() { return ""; }, set() { return ""; }})',
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+    },
+    {
+      code: 'let foo = computed("test", function() { return ""; })',
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+    }
+  ],
+  invalid: [
+    {
+      code: 'let foo = computed("test", function() { })',
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+      errors: [{
+        message: 'Return a value from computed property'
+      }]
+    },
+    {
+      code: 'let foo = computed("test", function() { if (true) { return ""; } })',
```

Comment:
why is this check invalid? it seems correct to me as there is a `return` statement inside the function 🤔

Reply:
@gmurphy any comments?

Reply:
Sorry for the long radio silence. Since not all paths `return`, I thought this should be an invalid case. I've added a valid case for this example as well.